templates/default/sql/dml/2.2.0.sql.erb

```diff
@@ -35,3 +35,8 @@ REPLACE INTO `hopsworks`.`variables`(`id`, `value`) VALUES ("fs_job_activity_tim
 
 REPLACE INTO `hopsworks`.`variables`(`id`, `value`) VALUES ("pypi_indexer_timer_interval", "<%= node['hopsworks']['pypi_indexer_timer_interval'] %>");
 REPLACE INTO `hopsworks`.`variables`(`id`, `value`) VALUES ("pypi_simple_endpoint", "<%= node['hopsworks']['pypi_simple_endpoint'] %>");
+
+INSERT INTO `hopsworks`.`users`
+(`username`, `password`, `email`, `fname`, `lname`, `activated`, `title`, `orcid`, `false_login`, `isonline`, `secret`, `validation_key`, `mode`, `password_changed`, `notes`, `status`, `max_num_projects`, `two_factor`, `tours_state` , `salt`) VALUES
+('onlinefs', '<%= @onlinefs_password %>', '<%= @onlinefs_email %>', 'OnlineFS', 'Server', '2021-03-16 16:17:00', 'Mr', '1234-5678-1234-5678', -1, 0, 'V3WBPS4G2WMQ53VA', NULL, 0, '2021-03-16 16:17:00', NULL, 2, 30, 0, 3, '<%= @onlinefs_salt %>')
+ON DUPLICATE KEY UPDATE `username`="onlinefs";
```

Comment:
Is there a single db user called 'onlinefs'?
Before we had  an online database per online feature store. You would have a different username/password for each online FS. Is this a single global onlinefs user?